deploy/helm/sumologic/templates/_helpers.tpl

```diff
@@ -0,0 +1,56 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "sumologic.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If dryRun=true, we use fixed value "fluentd".
```

Comment:
when we need to generate yaml files, we would set `dryRun` (for the lack of a better name) to true to use fixed values for release name and the app label
eg.
`helm template ./sumologic --set dryRun=true --namespace sumologic`

Reply:
this is such that the generated yaml for non-helm users will be consistent with the yaml we currently provide in the repo. for example our current yaml uses `fluentd` as the name for a lot of resources but in helm we would prefix with the release name followed by the chart name